Nos/Models/PublicKey.swift

```diff
@@ -37,26 +37,32 @@ struct PublicKey {
     }
     
     init?(npub: String) {
-        self.init(bech32Encoded: npub, prefix: Nostr.publicKeyPrefix)
+        self.init(bech32Encoded: npub, prefix: NostrIdentifierPrefix.publicKey)
     }
 
     init?(note: String) {
```

Comment:
The following `init` contains the bug fix, but you know what? I'm not sure `init?(note:)` should even exist. I'm working to remove it.

Reply:
It's gone!

Reply:
🎉 yay